tests/README.md

```diff
@@ -38,14 +37,13 @@ To use Libvirt backend, you need to have:
   - `virt-viewer`       (optional, to access graphical console of VMs)
   - `ansible`           (required if remediating via Ansible)
   - `edk2-ovmf`         (required if you want to install UEFI based machine)
-- A VM that fullfils following requirements:
+- A VM that fulfils the following requirements:
   - Package `qemu-guest-agent` installed
   - Package `openscap` version 1.2.15 or higher installed
   - `root` can login via ssh (it is recommended to setup key-based authentication)
   - `root` can install packages (for RHEL7, it means subscription enabled).
```

Comment:
One thing not mentioned is that the VM needs to have `CPE_NAME` in `/etc/os-release`. I came across this because Ubuntu doesn't ship such in a stock image. It isn't a lot of work to add, but it is something to note.

https://bugs.launchpad.net/ubuntu/+source/base-files/+bug/1472288 

It didn't get picked up for Hirsute, so it'll likely be Impish (21.10) or later (22.04 LTS).